.gitignore

```diff
@@ -112,4 +112,5 @@ travis
 helm-chart/travis-binder.yaml
 
 # Federation data page
-doc/federation/data-federation.txt
\ No newline at end of file
+doc/federation/data-federation.txt
+.vscode/
```

Comment:
nit: I'm not wild about having editor specific ignores. Mostly because I fear we end up with many many different (poorly maintained) ignores for all the different editors people use. I think it is nicer if each of us has their own "global" gitignore a la https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore